src/commonmark/en/content/user/analyze-data-in-pivot-tables.md

```diff
@@ -447,6 +447,13 @@ write is visible to those people.
     an internal DHIS2 message with the interpretation or comment. You
     can see the interpretation in the **Dashboard** app.
 
+    It is possible to format the text with **bold**, *italic* by using the
+    Markdown style markers \* and \_ for **bold** and *italic* respectively.
+    Keyboard shortcuts are also available: Ctrl/Cmd + B and Ctrl/Cmd + I. A
+    limited set of smilies is supported and can be used by typing one of the
+    following character combination: :) :-) :( :-( :+1 :-1. URLs are
```

Comment:
combination -> combinations

Same for all 4 locations

Reply:
Done, also for the source (managing-dashboards.md)